platform/fio/actor.go

```diff
@@ -0,0 +1,105 @@
+package fio
+
+import (
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/trustwallet/blockatlas/pkg/errors"
+)
+
+func actorFromPublicKeyOrActor(addressOrActor string) string {
+	len := len(addressOrActor)
+	if len >= 51 && len <= 55 && addressOrActor[:3] == "FIO" {
+		// assume public key string
+		pkBytes, err := bytesFromPublicKeyString(addressOrActor)
+		if err != nil {
+			return ""
+		}
+		return actorFromPublicKeyBytes(pkBytes)
+	}
+	if len <= 13 {
+		// assume actor
+		return addressOrActor
+	}
+	return ""
+}
+
+func actorFromPublicKey(address string) string {
```

Comment:
Why do we need that function? It is unused

Reply:
my mistake, supposed to be called from func above.  done.